test/idp/src/main/kotlin/org/codice/compliance/web/sso/PostLoginTest.kt

```diff
@@ -49,8 +50,12 @@ class PostLoginTest : StringSpec() {
                     .post(Common.getSingleSignOnLocation(SamlProtocol.POST_BINDING))
 
             response.statusCode shouldBe HTTP_OK
-            val idpResponse = getServiceProvider(IdpResponder::class.java).getIdpPostResponse(response)
-            val responseDom = verifyResponse(idpResponse)
+            val idpResponse = getServiceProvider(IdpResponder::class.java)
+                    .getIdpPostResponse(response).decorate()
+
+            idpResponse.bindingVerifier().verify()
+
+            val responseDom = idpResponse.responseDom!!
```

Comment:
❗️ This can potentially through an NPE (protection against which is, please recall, one of the primary advantages that Kotlin provides over Java).

See note on `IdpResponseDecorators.kt`.